docs/tutorials/index.md

```diff
@@ -22,34 +22,33 @@ title: "Quickstart (local)"
   ~ under the License.
   -->
 
-This quickstart gets you started with Apache Druid and introduces you to Druid ingestion and query features. For this tutorial, we recommend a machine with at least 6 GB of RAM.
+This quickstart helps you install Apache Druid and introduces you to Druid ingestion and query features. For this tutorial, you need a machine with at least 16 GiB of RAM.
 
-In this quickstart, you'll do the following:
+In this quickstart, you'll:
 - install Druid
 - start up Druid services
 - use SQL to ingest and query data
 
 Druid supports a variety of ingestion options. Once you're done with this tutorial, refer to the
 [Ingestion](../ingestion/index.md) page to determine which ingestion method is right for you.
 
-## Requirements
+## Prerequisites
 
 You can follow these steps on a relatively modest machine, such as a workstation or virtual server with 16 GiB of RAM.
 
 Druid comes equipped with launch scripts that can be used to start all processes on a single server. Here, we will use [`auto`](../operations/single-server.md#druid-auto-start), which automatically sets various runtime properties based on available processors and memory.
 
-In addition, Druid includes several [bundled non-automatic profiles](../operations/single-server.md) for a range of machine sizes. These range from nano (1 CPU, 4GiB RAM) to x-large (64 CPU, 512GiB RAM). 
-We won't use those here, but for more information, see [Single server deployment](../operations/single-server.md). For additional information on deploying Druid services across clustered machines, see [Clustered deployment](./cluster.md).
+In addition, Druid includes several [bundled static profiles](../operations/single-server.md) for a range of machine sizes. These range from nano (1 CPU, 4 GiB RAM) to x-large (64 CPU, 512 GiB RAM).
```

Comment:
Perhaps add, "Many people use the micro-quickstart profile when learning Druid." That way, folks won't have to guess which one to pick to get started.

Reply:
You don't need to pick one. `/bin/start-druid` should theoretically work without having to specify.